projects/plugins/jetpack/_inc/lib/core-api/load-wpcom-endpoints.php

```diff
@@ -42,8 +42,7 @@ function wpcom_rest_api_v2_load_plugin( $class_name ) {
 	global $wpcom_rest_api_v2_plugins;
 
 	if ( ! isset( $wpcom_rest_api_v2_plugins ) ) {
-		$wpcom_rest_api_v2_plugins             = array();
-		$_GLOBALS['wpcom_rest_api_v2_plugins'] = array(); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase,VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
Way back in #9968, for some reason someone did both `global $wpcom_rest_api_v2_plugins;` and tried to assign `$_GLOBALS['wpcom_rest_api_v2_plugins']`, which of course doesn't do anything useful since the superglobal is `$GLOBALS` with no underscore.

Not sure why we added the `phpcs:ignore` in #23557 rather than cleaning it up then.